qemu/tests/cfg/rh_kernel_update.cfg

```diff
@@ -43,6 +43,8 @@
     install_virtio = no
     verify_virtio = no
     virtio_drivers_list = "virtio virtio_ring virtio_pci"
+    !RHEL.7, RHEL.8:
+        virtio_drivers_list = ""
```

Comment:
Well, I have not use this case since rhel8.
Does rhel8 guest works ? 
My rhel8 guest env, it missed the defined driver too:
```
[root@vm-73-150 ~]# uname -r
4.18.0-293.el8.x86_64
# lsmod  | grep virtio
virtio_net             53248  0
net_failover           24576  1 virtio_net
virtio_blk             20480  3
```

Reply:
The status above is not for rh_kernel_update but the default vm with boot.
After check the whole rh_kernel_update file, i am afraid this is a product bug with the new product that `mkinitrd` does not work as expected. Need future checking.